.tool-versions

```diff
@@ -3,7 +3,7 @@ gcloud 448.0.0
 helm 3.13.0
 kapp 0.59.0
 kubectl 1.28.2
-python 3.11.5
+python 3.12.0
```

Comment:
We don't have a single python-file here. Why do we even need a python interpreter in the tool-chain?

Reply:
Adding info: This is needed by the tool gcloud!